README.md

```diff
@@ -7,19 +7,108 @@ watching and configuring their CustomResources (CRs).
 
 ## Build
 
+### OCS Operator
 The operator is based on the [Operator
 SDK](https://github.com/operator-framework/operator-sdk). In order to build the
 operator, you first need to install the SDK. [Instructions are
 here.](https://github.com/operator-framework/operator-sdk#quick-start)
 
 Once the SDK is installed, the operator can be built via:
 
-```bash
+```console
 $ dep ensure --vendor-only
 
 $ operator-sdk build quay.io/openshift/ocs-operator
 ```
 
+### Converged CSV
+
+_TODO: Add instructions to build the converged CSV and manifests_
+
+## Install
+
+The OCS operator can be installed into an Openshift cluster using the OLM.
+
+For quick install using pre-built container images, deploy the [deploy-olm.yaml] manifest.
+
+```console
+$ oc create -f ./deploy/deploy-with-olm.yaml
+```
+
+This will create a custom CatalogSource, a new openshift-storage Namespace, an
+OperatorGroup, and a Subcription to the OCS catalog in the openshift-storage
+namespace.
+
+Once this is done, a StorageCluster can be created.
+
+```console
+$ oc create -f ./deploy/crds/ocs_v1alpha1_storagecluster_cr.yaml
+```
+
+### Installation of development builds
+
+To install own development builds of OCS, first build and push the ocs-operator image to your own image repository.
+
+Once the ocs-operator image is pushed, edit the CSV to point to the new image.
+
+```
+$ OCS_OPERATOR_IMAGE="custom-ocs-operator-image:with-tag"
+$ sed -i "s|quay.io/ocs-dev/ocs-operator:latest|$OCS_OPERATOR_IMAGE" ./deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml
+```
+
+Then build and upload the catalog registry image.
+
+```console
+$ export REGISTRY_NAMESPACE=<quay-username>
+$ export CONTAINER_TAG=<some-tag>
+$ ./hack/build-registry-bundle.sh
```

Comment:
can we get a `make` target for this?

Also, would it make sense to separate the container build step from pushing the bundle container to a registry? For example, In my dev flow I don't always want to push the container to a registry.

Reply:
It doesn't look like there's any CI verifying this build step either. The script fails for me locally https://github.com/openshift/ocs-operator/issues/17

Separating the container build from pushing would allow us to make a simple job to verify container building works as well.